workflow/rocoto/workflow_tasks.py

```diff
@@ -516,13 +516,29 @@ def ocnanalprep(self):
 
         return task
 
-    def ocnanalrun(self):
+    def ocnanalbmat(self):
 
         deps = []
         dep_dict = {'type': 'task', 'name': f'{self.cdump}ocnanalprep'}
         deps.append(rocoto.add_dependency(dep_dict))
         dependencies = rocoto.create_dependency(dep=deps)
 
+        resources = self.get_resource('ocnanalbmat')
+        task = create_wf_task('ocnanalbmat',
+                              resources,
+                              cdump=self.cdump,
+                              envar=self.envars,
+                              dependency=dependencies)
+
+        return task
+
+    def ocnanalrun(self):
+
+        deps = []
+        dep_dict = {'type': 'task', 'name': f'{self.cdump}ocnanalbmat'}
```

Comment:
I think these two task names are reversed? `ocnanalprep` is under `ocnanalbmat` and `ocnanalbmat` is under `ocnanalrun`

Reply:
@guillaumevernieres Does any of this run on WCOSS?

Reply:
> @guillaumevernieres Does any of this run on WCOSS?

@AndrewEichmann-NOAA , I don't think we can build the JEDI system yet on WCOSS, so No.

Reply:
Oh, duh

Reply:
@guillaumevernieres Technically you can build on WCOSS2.  @CoryMartin-NOAA knows how.

Reply:
yes.  I don't think we are asking about WCOSS2.